Makefile

```diff
@@ -38,7 +38,7 @@ tidy:
 
 .PHONY: image-build
 image-build:
-	$(IMAGE_BUILDER) build -t $(IMAGE_REPO)/preflight:$(VERSION) .
+	$(IMAGE_BUILDER) build --build-arg release_tag=$(RELEASE_TAG) --build-arg preflight_commit=$(VERSION) -t $(IMAGE_REPO)/preflight:$(VERSION) .
```

Comment:
Non-blocking: I'd like to see these maybe passed in as a make envvar? Then they can be modified, added to, overridden from elsewhere (or even not present in a local build).